inc/storefront-template-functions.php

```diff
@@ -183,8 +183,8 @@ function storefront_site_branding() {
 	 */
 	function storefront_primary_navigation() {
 		?>
-		<nav id="site-navigation" class="main-navigation" role="navigation" aria-label="<?php esc_html_e( 'Primary Navigation', 'storefront' ); ?>">
-		<button class="menu-toggle" aria-controls="primary-navigation" aria-expanded="false"><span><?php echo esc_attr( apply_filters( 'storefront_menu_toggle_text', __( 'Menu', 'storefront' ) ) ); ?></span></button>
+		<div id="site-navigation" class="main-navigation" role="navigation" aria-label="<?php esc_html_e( 'Primary Navigation', 'storefront' ); ?>">
```

Comment:
> I also think we should include the header

You just decided to remove headings from sidebars for you know which good reason... :P

> Valid html feels good.

Agreed, but we could argue about what is "valid HTML"... If you did read my line note about the `role` attribute, even W3C doesn't know. 😆 
Also, `<nav>` element without heading actually do **not** even throw a warning on Nu HTML Checker.

> It's good for screen readers considering we have multiple navs

There is nothing else to do for helping them ? We can't help them differentiate `<nav>` with `aria*` attributes ? I don't know much about that...